tao/build_defs.bzl

```diff
@@ -0,0 +1,28 @@
+def tao_bridge_cc_test(
+        name,
+        srcs,
+        deps = [],
+        data = [],
+        linkstatic = 0,
+        copts = [],
+        linkopts = [],
+        **kwargs):
+    native.cc_test(
+        name = name,
```

Comment:
Arguments like `name`/`data`/`srcs` are just passed as it is, why not just leave them in `kwargs`